test.sh

```diff
@@ -56,6 +56,31 @@ function test_metrics() {
 
 # ==================================================================================
 
+function test_entrypoint() {
+  local name=$1
+  local entrypoint=$2
+
+  local container_id=$(docker run --name ${name}-test -d \
+                         -e LANG=en_US.UTF-8 -e PARAMETER_THAT_MAY_NEED_ESCAPING="&'\"|< é\\(" ${name} \
```

Comment:
I'm curious about the use of `-e LANG=en_US.UTF-8` here.. is it required just for this particular test, or should be added in general??

Reply:
UTF-8 is requieredto pass in arguments with special characters. if you renove it then the test will fail. I'm not sure but I guess its because the test JAR (as 99% of all Jars) was build with UTF-8 encoding)